### ISSUE TITLE
README.md not in English language

### ISSUE THREAD

**Issue by dominiksalvet** [RichardLitt/standard-readme#81]
I have a repository where I use my native language, Czech language, [uvod_do_vhdl](https://gitlab.com/dominiksalvet/uvod_do_vhdl) - Introduction to VHDL. I use English only when necessary and to educate readers in good programmer's conventions.

The point is that I want to make the README.md file of the project to be Standard-Readme compliant, but by the specifications stated in the [spec.md](https://github.com/RichardLitt/standard-readme/blob/master/spec.md) file I can't as the titles aren't the same in those languages obviously. Currently, I am using direct translations of the titles from English to Czech. The README.md file should otherwise be Standard-Readme compliant.

Can the README.md file at it's current state be declared as a Standard-Readme compliant one? If not, how to comply both the czech readers and Standard-Readme, please?

Thanks.

**Comment by RichardLitt** [comment]
Yes, I would certainly call that compliant. However:

- The file is named README.md. We should add a note that if it is the only README, it does not need to be named `README-cs.md`. 
- We should add a note that subject headings should be titled in the relevant language. This would match the language mentioned in the file title, unless it is the only README (as in your case), in which case it is assumed that it would be in Czech. 

How do these two small fixes to SR sound to you?

**Comment by dominiksalvet** [comment]
Well, I was thinking about naming the *README.md* file before opening this issue. And, by specification, it is required to name it like *README-cs.md* only if a project supports i18n localization, which isn't really the case in my case because I don't even know what it is. :) And so I interpreted it like it is not required to name it like stated.

Your suggestion about naming README files with BCP 47 language tag only when more than one README file exist it's quite fair. The second solution that immediately comes up to my mind is to name README files with the tag only when a different language than English is used. However, I don't know if it's as universal as your solution. In my opinion, your solution could be slightly better.

About the titles; two extreme solutions bring those characteristics:
* When used only English, the titles are exactly defined and user, who knows what "Standard-Readme" means, can quickly get through a compliant README file. However, README files written in a different language (with English titles) can be problematic for target readers.
* When used only a target language, the titles are not exactly defined (and when not used English, the titles can be nearly completely arbitrary as more translations exist). Additionally, it doesn't make sense to maintain a template for every language just to define the titles. However, those README files may be good for target readers.

I am thinking about something like that (it still keeps at least some of the English README structure):
> Přispět (Contribute)

But I'm still not decided if it's a good solution. On the other hand, your suggestion relies more on people - use just a target language without an English word as it can confuse the readers. In my opinion, your suggested solutions is better as long as people will use some direct-translated titles from the original ones.

So basically, I agree with you, but you have the final word.

**Comment by RichardLitt** [comment]
Alright! Patched. Thank you for your help, @dominiksalvet. 🎉
